tests/code/test_users.py

```diff
@@ -322,3 +321,27 @@ def test_getUserBGCheckHistory():
 
 
         transaction.rollback()
+
+@pytest.mark.integration
+def test_getBannedUsers():
+    with mainDB.atomic() as transaction:
```

Comment:
Remove this as it is unnecessary, please,

Reply:
Done